generators/cleanup.js

```diff
@@ -75,6 +77,29 @@ function cleanupOldFiles(generator) {
         generator.removeFile(`${ANGULAR_DIR}admin/metrics/metrics-modal.component.ts`);
         generator.removeFile(`${CLIENT_TEST_SRC_DIR}spec/app/admin/metrics/metrics-modal.component.spec.ts`);
     }
+    if (generator.isJhipsterVersionLessThan('6.3.0')) {
```

Comment:
I am not sure if upgrade works correctly at moment. Yesterday, I was not able to get pass this check with `6.3.0` as version was already upgraded and this condition always evaluated to `false`. Were you able to test upgrade?

Reply:
I tested manual upgrade and that worked for me. Did you test manual upgrade or `upgrade` sub-generator?

Reply:
I am able to reproduce the problem. I updated `package.json`, set `version` to `6.3.0` and no files were deleted. Will investigate this.

Reply:
I think that's not an issue with your change. To test your change, you can use version as `6.4.0` in this condition and `6.3.0` in `generator-jhipster` package.json (please note this is temporary change to test and shouldn't commit)

Reply:
If I tested such way then files were deleted correctly. But I'm not sure if this PR can be merged while this problem is not solved.

Reply:
I think that's separate issue and this PR can move ahead